src/backend/distributed/worker/worker_merge_protocol.c

```diff
@@ -81,6 +81,7 @@ worker_merge_files_into_table(PG_FUNCTION_ARGS)
 	/* we should have the same number of column names and types */
 	int32 columnNameCount = ArrayObjectCount(columnNameObject);
 	int32 columnTypeCount = ArrayObjectCount(columnTypeObject);
+
```

Comment:
Oops, it remained from a change I revert back.  I will fix this with the test change.